src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import { UserInfo } from '../UserInfo';
+import '../../App.scss';
+import { Post } from '../../types/post';
+import React from 'react';
+
+type TodoInfoProps = {
+  todo: Post;
+};
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo }) => {
+  const { id, user, title, completed } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={`TodoInfo ${completed ? 'TodoInfo--completed' : ''}`}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+
+      {user && <UserInfo user={user} />}
```

Comment:
[REACT KNOWLEDGE] - This is a minor issue, but it's better to avoid rendering the component if the property that you pass to the component has `null` or `undefined` value. In this case, you could handle the `null` or `undefined` value for `user` outside of the `UserInfo` component.